plugins/wp-gatsby-gutenberg/lib/src/preview/gatsby.js

```diff
@@ -0,0 +1,78 @@
+import { getSaveContent } from "@wordpress/blocks"
+import { debounce } from "lodash"
```

Comment:
The same goes for above. So its actally contraproductive to import `lodash.debounce` instead of lodash since it would be treted as non external and bundled twice with lodash, which is available in window.lodash itself inside wp.

Reply:
Yeah, i have checked the docs and its actually best not to require those packages inside package.json. Code completion works anyways